packages/helpers/src/input-generators.ts

```diff
@@ -12,13 +12,16 @@ type CircuitInput = {
   emailBodyLength?: string;
   precomputedSHA?: string[];
   bodyHashIndex?: string;
+  mask?: number[];
```

Comment:
shouldn't this be a bit type?

Reply:
Good catch! I think we can add this in the masking circuit itself, by checking if all the mask elements are bit or not, rather than changing here.

Reply:
er well it doesn't really matter what it is in the circuit right, that doesn't need to be constrained there since it'll be public anyways. i think the type should be made clear here for devs?

Reply:
There's also a security aspect yeah, you can give a particular mask to manipulate the body and prove some wrong body content. Asserting that each mask element is a body prevent that and ensures that only the body content is hidden and not manipulated.

Reply:
Anyway I've fixed it